### ISSUE TITLE
Unable to drag n drop cards across stacks when there's a horizontal scrollbar

### ISSUE THREAD

**Issue by bsavant** [joplin/plugin-yesyoukan#11]
If there are enough stacks on the board to create a horizontal scroll, it doesn't allow you to properly drag and drop across them.

https://github.com/user-attachments/assets/9a8b56b5-66bf-486c-89d6-e39141951853

**Comment by laurent22** [comment]
I can't get it to scroll when I drag a card, so for me it kind of works because wherever I'll be able to drop it, it will be valid. Is that the latest version of the plugin v1.0.10?

I think not being able to scroll while dragging would be an acceptable fix.

**Comment by bsavant** [comment]
Yeah, I am using the latest v1.0.10.  I'm on a MacBook Pro using the trackpad I can hold the card and scroll seamlessly.

**Comment by ernstki** [comment]
@bsavant Wow, this might be a tough one for Laurent to fix. I wouldn't be surprised if it's an Electron problem, or something to do with the actual rendering engine itself.

I'm using the latest (development) release of Joplin on a Linux system with a multi-touch capable trackpad, and I'm able to reproduce this behavior, **only** because I'm able to trigger a horizonal scroll with extra fingers while keeping the button depressed for the drag operation. So I _can_ confirm that it's not just specific to your setup or the OS.

**Comment by bsavant** [comment]
Thanks for confirming @ernstki .  I went back and tested this on an older version of Joplin(v2.14.20) where I could get the original [Kanban plugin](https://github.com/joplin/plugin-kanban) to work, and that one does function across horizontal scrolls if that's of any help.

**Comment by laurent22** [comment]
I expect this bug won't be fixed any time soon. The current dnd lib I use doesn't support this and unfortunately is now deprecated so it won't be added. The alternative lib I could switch to support this but doesn't look as nice when drag and dropping objects. So let's see anyway if there's much of a need to support this - I consider a bit that it's a niche usage at this point.

I'm putting here the alternative lib for reference:

https://atlassian.design/components/pragmatic-drag-and-drop/optional-packages/react-beautiful-dnd-migration/about

I tried it and it works but as mentioned it doesn't look as nice.

**Comment by bsavant** [comment]
Out of my own curiosity, I used a board with a large number of stacks and reorganized the stacks themselves through a horizontal scroll.  I would have expected that this might have some issues too, but it works perfectly.  Is the implementation of the draggable cards functionally different than the stacks?

**Comment by laurent22** [comment]
Yes, scrollable areas within scrollable areas are not properly supported

**Comment by laurent22** [comment]
Closing for now. Let's see if we can switch to a different lib at some other time
